state/account_test.go

```diff
@@ -175,3 +175,45 @@ func TestClone(t *testing.T) {
 	require.Equal(big.NewInt(200), ss.Balance)
 	require.Equal(big.NewInt(200+100), account.Balance)
 }
+
+func TestConvertFreshAddress(t *testing.T) {
```

Comment:
e2etest is also required to check working well in evm

Reply:
added in latest commit

Reply:
added in latest commit